Resources/Prototypes/Catalog/VendingMachines/snack.yml

```diff
@@ -4,9 +4,8 @@
   animationDuration: 0.5
   spriteName: snack
   startingInventory:
-    FoodSpaceTwinkie: 8
```

Comment:
Was this meant to be removed?

Reply:
Must have missed it off the sort. Fixed thanks.